projects/Re-Isearch/test.nix

```diff
@@ -0,0 +1,98 @@
+{
+  sources,
+  lib,
+  ...
+}:
+let
+  docDir = "documents";
+  docPath = filename: "${docDir}/${filename}";
+  docs = [
+    # PDF support not functional, needs custom pdftomemo from /filters/xpdf-3.01-bsn to be built & installed
```

Comment:
I have a note here that PDF support doesn't work & needs the seemingly custom `pdftomemo` from the source. Do you want me to expand this with the info that it needs a `re-isearch.src` newer than what's in Nixpkgs to appear there, and the questionable security of that project?

Reply:
Yes, essentially what you wrote in the PR description

Reply:
Expanded the comment with more information.